.github/workflows/release.yml

```diff
@@ -2,9 +2,8 @@ name: Release
 on:
   push:
     branches:
-      - master
-      - next
-      - beta
+      - main
```

Comment:
- Move from `master` to `main` on this fork
- `develop` may be used to keep two repos in sync until moving to `main`